app/components/widgets/forms/social-link-field.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="{{unless @fixed 'fields'}}">
   {{#unless @fixed}}
-    <div class="three wide field">
+    <div class="four wide field">
```

Comment:
I don't understand how is this a correct solution.
Changing the width of the element wont fix the problem with the spacing in between elements. At some point on reducing the width (which you could have tested on your local) this would again overlap.
The issue is with the margin in between the elements and not with the width of either elements.
Infact now this makes it even worse on the tablet view.

Reply:
Yes, sorry I didn't checked for different resolution, surely this  won't fix for that.